usaspending_api/common/recipient_lookups.py

```diff
@@ -6,10 +6,9 @@ def obtain_recipient_uri(
         recipient_unique_id,
         parent_recipient_unique_id=None,
         is_parent_recipient=False):
-    # No need to create a hash when the parent recipient does not exist
-    if is_parent_recipient and not recipient_unique_id:
```

Comment:
Was this parent logic not important? I didn't see it in the ticket description.

Reply:
Previously I added this logic to prevent altering the way that the functionality for the recipient hash worked and only affects the parent recipient. There is a bug where even the recipient hash runs into an issue when there is not an id and name so I opened this up to return None for the hash regardless of parent or not.